### PR TITLE
:loud_sound: optimize log and not assert the container must be exist after micro app unmounted

### PR DESCRIPTION
**Pull request by kuitos**
This change is [Reviewable](https://reviewable.io/reviews/umijs/qiankun/495)

### CODE REVIEW

File: src/loader.ts [hunk 1]
```diff
@@ -107,13 +123,26 @@ function getRender(appContent: string, container?: string | HTMLElement, legacyR
...
-    );
 
-    if (!containerElement!.contains(element)) {
+    // The container might have be removed after micro app unmounted.
+    // Such as the micro app unmount lifecycle called by a react componentWillUnmount lifecycle, after micro app unmounted, the react component might also be removed
+    if (phase !== 'unmounted') {
+      const errorMsg = (() => {
+        switch (phase) {
+          case 'loading':
+            return `[qiankun] Target container with ${container} not existed while ${appName} loading!`;
```
Comment (Deturium) [inline src/loader.ts:133]: 这几个 case 可以合并成 
`[qiankun] Target container with ${container} not existed while ${appName} ${phase}!` 的吧

File: src/loader.ts [hunk 2]
```diff
@@ -85,17 +92,26 @@ function getAppWrapperGetter(
 
 const rawAppendChild = HTMLElement.prototype.appendChild;
 const rawRemoveChild = HTMLElement.prototype.removeChild;
-type ElementRender = (props: { element: HTMLElement | null; loading: boolean }) => any;
+type ElementRender = (
+  props: { element: HTMLElement | null; loading: boolean },
```
Comment (Deturium) [inline src/loader.ts:96]: 这个改动会影响使用 `legacyRender` 的未更新用户吧？

Comment (kuitos) [reply src/loader.ts:96]: 为啥？

Comment (kuitos) [reply src/loader.ts:96]: 这是个内部类型，为啥会影响到外部用户？

File: src/loader.ts (hunk 1 above)
Comment (kuitos) [reply src/loader.ts]: done

**Pull request merged**